NEWS.md

```diff
@@ -20,6 +20,7 @@ The new features include:
 ## Minor changes
 
 * The column name for both `languagecodes` and `countrycodes` is now `code`, and not `alpha2` and `Code`, respectively. 
+* HTTP error messages are returned directly from the API, not as a canned response from `code_message`. The latter has therefore been deleted. 
```

Comment:
maybe add a tiny bit of context i.e. the messages from the API are more informative?

Reply:
The messages are shorter actually: https://github.com/ropensci/opencage/issues/69#issuecomment-491495035
See e.g. the 403 status message above.